web/components/ui/Header/Header.tsx

```diff
@@ -57,9 +59,10 @@ export const Header: FC<HeaderComponentProps> = ({ name, chatAvailable, chatDisa
         <UserDropdown id="user-menu" hideTitleOnMobile showToggleChatOption={canHideChat} />
       )}
       {!chatAvailable && !chatDisabled && (
-        <Tooltip
+        
+       <Tooltip
           overlayClassName={styles.toolTip}
-          title="Chat will be available when the stream is live."
+          title={!online?"Chat will be available when the stream is live.":"Chat disabled by streamer"} 
```

Comment:
I think if chat is disabled, we shouldn't say anything. The user can't do anything about it, and it's probably just confusing if they didn't know chat was a feature in the first place.